addons/skin.confluence/language/English/strings.po

```diff
@@ -284,16 +293,17 @@ msgstr ""
 
 #empty string with id 31133
 
+#. In main language strings its used as submenu
 msgctxt "#31134"
-msgid "Home page Videos sub-menu"
+msgid "Home page videos submenu"
```

Comment:
"Videos", "Music", "Pictures" in these labels seem to be references to the according main menu sections?